### ISSUE TITLE
Can't add facsimile from a parent Github repo

### ISSUE THREAD

**Issue by teymuri** [mei-friend/mei-friend#100]
I can't get mei-friend to show a facsimile: I am logged into a github repositor, with an mei file and an image in it. Trying to  add a  facsimile element with Manipulate->Add facsimile element and changing the target to the jpg file:

```
<facsimile xml:id="fd8jwm01">
         <surface xml:id="s0wsox2">
            <graphic xml:id="g7puxax1" target="test.jpg" width="500" height="500"/>
         </surface>
      </facsimile>
``` 

Does not show the jpg file, the verovio and facsimile Logo's spin for ever and nothing happens. Is there any way to find out why this approach is not work?

**Comment by wergo** [comment]
Could you please provide the full MEI file?

(You have to reference the `surface` element from somewhere inside the `body`, for instance through a pb@facs or any other element.)

**Comment by teymuri** [comment]
> Could you please provide the full MEI file?
> 
> (You have to reference the `surface` element from somewhere inside the `body`, for instance through a `pb@facs` or any other element.)

Thank you for your fast reply!
Here is the mei file: I have supplied the surface xml:id to a pb element with the same id. But the image is still not showing in the facsimile window. Here is the mei file: https://github.com/teymuri/molter/blob/main/test.mei

**Comment by wergo** [comment]
Thanks a lot @teymuri for the file! The problem has been caused by one `@facs` attribute pointing to the non-existent zone id `zvab1i21`: 

```xml
<rest xml:id="r1dppjr5" dur="4" dur.ppq="2" facs="#zvab1i21"/>
```

mei-friend is now able to handle those cases gracefully (fixed in 2121cfb4a53ee76e80c76c5894fc72e305bda80a). The latest version is available at our staging environment. Please test it on your system: https://staging.mei-friend.mdw.ac.at?file=https://raw.githubusercontent.com/teymuri/molter/main/test.mei.
